packages/split-button/src/SplitButton.ts

```diff
@@ -186,26 +186,26 @@ export class SplitButton extends SizedMixin(PickerBase) {
         }
     }
 
-    protected manageSelection(): void {
+    protected async manageSelection(): Promise<void> {
         super.manageSelection();
```

Comment:
If changing the signature to be async, should this also await its super call to be sure order is preserved? (1. whatever super does, 2. this.manageSplitButtonItems())

```suggestion
        await super.manageSelection();
```

Reply:
Actually 👎🏼 ... I tried this and then it fails VRT because it updates the timing of the open/select/highlight code. It _likely_ points to some lucky timing vs intentional timing, but I'd like to skip digging into it while it doesn't show as broken in any of our tests/demos.

Reply:
Interesting, then I think we're running `super.manageSelection()` and `this.manageSplitButtonItems()` concurrently, and relying on some relative timing between the two.

Not changing while things pass works for me.

I'm curious what happens with this, just to complete all the different ways they could be ordered:

```ts
await this.manageSplitButtonItems()
await this.manageSelection()
```

Reply:
Trying in CI now. Having `manageSplitButtonItems` should encapsulate it in the asynchrony of `manageSelection` 🤞🏼 Hopefully, there's not other side effects here...